environment.yml

```diff
@@ -63,6 +63,8 @@ dependencies:
   - pytest-xdist>=1.31
   - pytest-asyncio
   - pytest-instafail
+  #redis IO testing
```

Comment:
space after `#`

Reply:
Resolved in commit 5c16ad6a001c1d11ffc911f70a491c59d6646e20